CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # CHANGELOG
 
-v0.9.2 - _Aug 21, 2017_
+
+v0.9.3 - _Aug 21, 2017_
 ------------------------
+    * Added good error message when checksummed address is submitted (#124)
```

Comment:
`Add clear error message when checksummed address is passed to a public method (#124)`